amulet/level/formats/anvil_world/format.py

```diff
@@ -391,6 +392,48 @@ def _get_raw_chunk_data(
         """
         return self._get_dimension(dimension).get_chunk_data(cx, cz)
 
+    def get_players(self) -> Generator[str, None, None]:
```

Comment:
I would prefer if this was `all_player_ids` so that it isn't confused with `get_players` and to match the `all_chunk_coords` method.